### PR TITLE
Embedded player for 3Speak

### PR DESCRIPTION
**Pull request by quochuy**

### CODE REVIEW

File: src/shared/HtmlReady.js [hunk 1]
```diff
@@ -87,7 +87,10 @@ export default function(html, { mutate = true, hideImages = false } = {}) {
     state.images = new Set();
     state.links = new Set();
     try {
-        const doc = DOMParser.parseFromString(html, 'text/html');
+        const doc = DOMParser.parseFromString(
+            preprocessHtml(html),
```
Comment (roadscape) [inline src/shared/HtmlReady.js]: Why is this needed if `embedThreeSpeakNode` is called in `linkifyNode`?

**Comment by quochuy** [comment]
Linkify node only allows transforming plain Url to an embedded player. 3speak however creates posts that contains an linked image, this allows to replace that image with a player:
https://steemit.com/hive-100421/@wehmoen/xrhjxocx

File: src/shared/HtmlReady.js (hunk 1 above)
Comment (quochuy) [reply src/shared/HtmlReady.js]: Linkify node only allows transforming plain Url to an embedded player. 3speak however creates posts that contains an linked image, this allows to replace that image with a player:
https://steemit.com/hive-100421/@wehmoen/xrhjxocx

Additionally, linkify is not being called because the HTML image tag is wrapped inside a CENTER tag, which is why I needed to preprocess the HTML first

**Comment by roadscape** [comment]
Local PR: #3655
